src/Styleguide/Utils/Responsive.tsx

```diff
@@ -2,6 +2,13 @@ import React from "react"
 
 const ResponsiveContext = React.createContext({})
 
+const shallowEqual = (a, b) => {
+  for (let key in a) {
+    if (a[key] !== b[key]) return false
+  }
+  return true
+}
```

Comment:
https://lodash.com/docs/4.17.10#isEqual

Reply:
Yeah, I'm just wanting to keep this component clean so it could be released separately.

Reply:
Yeah this would make a great lib 👍